Palindrome.UnitTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+// Added a test for Empty Input.
+// Created branch 'review' for PR sharing.
+
+
+namespace PalindromeArray.UnitTests
+{
+    [TestClass]
+    public class PalindromeUnitTests
+    {
+        [TestMethod]
+        public void Check_Empty_Input()
+        {
+            // Arrange
+            Palindrome check = new Palindrome();
+            char[] arrToCheck = check.PalindromeHelper("");
+            // Act
+            bool boolResultOfPalindromeCheck = check.IsArrayPalindrome(arrToCheck);
+            // Assert
+            Assert.AreEqual(false, boolResultOfPalindromeCheck);
+        }
+        [TestMethod]
+        public void Check_Longest_Palindrome_In_Dictionary()
```

Comment:
Is there a reason (based on the code) that you need to test the longest word?

Reply:
Not at all, wikipedia got me reading about palindromes. Will remove.

Reply:
dont remove it. Its another test isnt it? you did the work to make it, id keep it.